group03.txt

```diff
@@ -0,0 +1,10 @@
+duclair
+idrissa
+valdes
+philomene
+babara
+jonathan
+cyprien
+nigieu
+pat
```

Comment:
why cann't put full name

Reply:
last name before first name

Reply:
It's not nigieu but Mr Nigieu

Reply:
Capital letters?

Reply:
my name is actually Lesly not Leslie lol